pontoon/teams/views.py

```diff
@@ -168,6 +168,7 @@ def ajax_permissions(request, locale):
     locale = get_object_or_404(Locale, code=locale)
     project_locales = locale.project_locale.visible().visible_for(request.user)
 
+    community_builder_level = 0
```

Comment:
Same as above - this variable does not actually contain the value of the community builder badge level per, and the variable name should reflect that.